src/matching.js

```diff
@@ -355,26 +356,24 @@ function buildGraph(matches, people) {
       }
 
       if (!matched) {
-        graph.push([i, j]);
+        graph.push([i + 1, j + 1]);
         edges++;
       }
     }
   }
-  graph["n"] = count - 1;
+  graph["n"] = count;
   graph["e"] = edges;
-
-  return graph;
 }
 
 function matchPeople(matches, people) {
-  graph = buildGraph(matches, people);
+  buildGraph(matches, people);
   let n = readGraph();
   let g = findMaximumMatching(n);
 
   let m = [];
 
   for (var i = 0; i < n; i++)
-    for (var j = i + 1; j < n; j++) if (g[i][j] == 2) m.push([i + 1, j + 1]);
+    for (var j = i + 1; j < n; j++) if (g[i][j] == 2) m.push([i, j]);
```

Comment:
why not just push the IDs here?

```suggestion
    for (var j = i + 1; j < n; j++) if (g[i][j] == 2) m.push([people[i].id, people[j].id]);
```